### DISCUSSION TITLE
migration not found (?)

### DISCUSSION THREAD

**Discussion by bandinopla** [prisma/prisma#14737] [Q&A]
im trying to setup prisma migrate on a database already populated with data.

**1.**
I did prisma db pull to create the schema from it
when running `prisma migrate dev --name initial-migration --create-only` i get a warning asking to RESET the database... i dont want that. So...

**2.**
Run this command `prisma migrate diff --from-empty --to-schema-datamodel prisma/schema.prisma --script > migration.sql` and manually move the sql file to a manually created `prisma/migrations/initial` folder

**3.**
Now i try to baseline that migration running `prisma migrate resolve --applied "initial"`

i get this error saying the migration cant be found.. but it's there and even prisma knows it.,..

![image](https://user-images.githubusercontent.com/71508858/183888929-f77e555f-9834-4a51-8da5-2dc601a1b368.png)

**Comment by austincrim** [comment]
Hey @bandinopla 👋,

What's the exact name of the folder in which you placed `migration.sql`? Is it `init` or `initial`?

**Reply by bandinopla** [reply]
initial. I wrote "init" on the original post but it was as you see on the screenshot of the console. As you can see prisma knows a migration called "initial" exists... it's weird.

**Reply by austincrim** [reply]
That does seem weird.

Would you mind opening a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with all your relevant environment info?

**Reply by bandinopla** [reply]
bug reported [issue 14762](https://github.com/prisma/prisma/issues/14762)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
